README.md

```diff
@@ -1,11 +1,14 @@
 # OSIE
 
 [![Build Status](https://drone.packet.net/api/badges/tinkerbell/osie/status.svg)](https://drone.packet.net/tinkerbell/osie)
+![](https://img.shields.io/badge/Stability-Maintained-green.svg)
 
 OSIE is the Operating System Installation Environment.
 It consists of an Alpine Linux based netboot image which fetches a prebuilt Ubuntu 16.04 container that does the actual installation.
 All of the above is built from this repository using `GNU Make`.
 
+This repository is [Maintained](https://github.com/packethost/standards/blob/master/maintained-statement.md) meaning that this software is supported by Packet and its community - available to use in production environments.
```

Comment:
Does this applies to Tinkerbell? :) If the idea is to keep the project not that tight to Packet?

Reply:
You're correct; that needs to say Tinkerbell, not Packet. Please hold.

Reply:
Signed-off-by: K Rain Leander <rainleander@users.noreply.github.com>